Early Access for workspace-classes and usage-based billing (#2715)
gitpod-io/website: gitpod/docs/workspace-classes.md

# {title}

> > {title} are currently in [Early Access](/docs/references/gitpod-releases) with [Usage-based Billing](/docs/usage-based-billing). Please [contact us](/contact/support) for an invitation, or [send feedback](https://github.com/gitpod-io/gitpod/issues/12473).

Reviewer: **question**: Maybe unnecessary, but what do you think of rephrasing this so that we lower the expectations on getting an invitation? Otherwise, this could be misinterpreted by users.

```suggestion
> {title} are currently in [Early Access](/docs/references/gitpod-releases) with [Usage-based Billing](/docs/usage-based-billing). Please [contact us](/contact/support) to request access, or [send feedback](https://github.com/gitpod-io/gitpod/issues/12473).
```

Workspace classes allow you to select the resources available to your workspaces.

On gitpod.io, the options are:

- **Standard**: up to 4 cores, up to 8GB RAM, 30GB storage

> - **Large**: up to 8 cores, up to 16GB RAM, 50GB storage

Reviewer: **praise**: Nice and standing out list! ✨

> You can configure the workspace class that should be used for your workspaces in your user preferences.

Reviewer: **issue(non-blocking)**: Just cross-posting a disagreeing perspective from https://github.com/gitpod-io/website/pull/2657#discussion_r961697701 on including images in our docs as this work-in-progress already includes obsolete screenshots below—see the updated settings left sidebar in preferences below,
